examples/with-stencil/packages/web-app/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "web-app",
+  "name": "stencil-web-app",
```

Comment:
I think this one doesn't need to change.

Reply:
It uses the same name as https://github.com/vercel/next.js/blob/0729394ee18f978e77c4792b4982fe10cb0a8b74/examples/with-yarn-workspaces/packages/web-app/package.json#L2 so for my tests I needed them to be different

Reply:
@merceyz It should be `with-stencil` 👍

Reply:
Updated it to `with-stencil-app`, can't have it conflict with the root